### ISSUE TITLE
OpenMP not found on macOS

### ISSUE THREAD

**Issue by drewitz** [rgugliel/CoxIter#19]
cmake doesn't find OpenMP on macOS.

**Comment by rgugliel** [comment]
Can you provide the version of macOS? And eventual other relevant information (e.g., whether OpenMP was installed or not, ...)

**Comment by drewitz** [comment]
Currently I'm working on Catalina.

A first fix might be in drewitz/CoxIter@8a779d83fbe0d4286ee98387.
It does compile with OpenMP but it seems to be slower for small graphs.
Also need to install it via `brew install libomp`.

**Comment by drewitz** [comment]
Also without OpenMP it compiles perfectly fine. It just might be nice to have it actually running in parallel and faster also on macOS.

**Comment by drewitz** [comment]
I tested it on Linux and cleaned up the changes I made to `CMakeLists.txt`:
drewitz/CoxIter@416c787ff260ae4643538945

OpenMP should be optional now. This could be reflected in the docs?

This fix is based on fix #16 (to be merged).

**Comment by rgugliel** [comment]
Solved by #16 , thanks!

**Comment by drewitz** [comment]
I just tried to install it on a Mac from scratch and the master branch didn't find openMP.
It is not already solved by #16 but by drewitz/CoxIter@416c787 (will open a new pull request in a minute)
